portfolio/src/main/webapp/script.js

```diff
@@ -48,3 +48,9 @@ function addRandomCat() {
   if (prev !== undefined)  catContainer.replaceChild(catimg, prev);
   else catContainer.appendChild(catimg);
 }
+
+const postId1 = "0cb628857f3c4c77bf7f9a879a6ec21d";
+fetch("https://potion-api.now.sh/html?id=" + postId1)
+    .then(res => res.text())
+    .then(text => { document.getElementById("post1").innerHTML = text; 
+});
```

Comment:
(1) does this have to be in the global scope? Could we have this inside a window.onload function, and that way the 'fetch' call happens once the window is loaded in javascript. 

(2) I realized with github i couldnt comment on unchanged lines, but just to make it more consistent javascript programming for now and the future, let's start removing the 'function' keywords in the javascript, and use arrow functions instead. This will make it more consistent, and more inline with the general javascript/typescript conventions most people use for best practice.
see style guide if you want to read more about this https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions.

(3) Structure the javascript files so all the constants are at the top

ie

```
// Imports from other files/libraries/ etc if you're using them, and depending on what 'framework' you might/might not use goes at the top.
// const lib1 = import/requirefromlib1

// Consts in one place
const CAT_IMAGES = []
const GREETING_CHOICES = []
const POST_ID = []

// Add the functions below, with the comment for each of the fn's.
window.onload = () => {
...
}
```

Reply:
I think I covered all of these!